addons/sale/migrations/13.0.1.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,115 @@
+# DONE: Post-migration: Updated relation table with new ids
+
+sale         / utm.campaign             / company_id (many2one)         : NEW relation: res.company, hasdefault
+# NOTHING TO DO: new feature with default value
+
+---XML records in module 'sale'---
+DEL crm.team: sales_team.team_sales_department (noupdate)
+# NOTHING TO DO: It's from sales_team module
+
+NEW ir.actions.act_window: sale.action_sale_order_generate_link
+DEL ir.actions.act_window: sale.action_open_sale_onboarding_quotation_layout
+NEW ir.actions.server: sale.model_sale_order_action_quotation_sent
+NEW ir.config_parameter: sale.default_confirmation_template (noupdate)
+# NOTHING TO DO: noupdate=0 or new xmls
+
+NEW ir.model.access: sale.access_account_move_manager
+NEW ir.model.access: sale.access_account_move_salesman
+DEL ir.model.access: sale.access_account_invoice_manager
+DEL ir.model.access: sale.access_account_invoice_salesman
+# DONE: pre-migration (renamed)
+
+NEW ir.model.access: sale.access_account_account_tag_sale_salesman
+NEW ir.model.access: sale.access_account_account_type_sale_salesman
+NEW ir.model.access: sale.access_account_partial_reconcile_salesman
+DEL ir.model.access: sale.access_account_invoice_line_salesman
+DEL ir.model.access: sale.access_account_invoice_tax_sale_manager
+DEL ir.model.access: sale.access_account_invoice_tax_salesman
+DEL ir.model.access: sale.access_product_price_history_salemanager
+NEW ir.rule: sale.account_move_line_personal_rule (noupdate)
+NEW ir.rule: sale.account_move_line_see_all (noupdate)
+NEW ir.rule: sale.account_move_personal_rule (noupdate)
+NEW ir.rule: sale.account_move_see_all (noupdate)
+NEW ir.ui.view: sale.account_invoice_view_tree
+NEW ir.ui.view: sale.assets_tests
+NEW ir.ui.view: sale.product_template_sale_form_view
+NEW ir.ui.view: sale.report_assets_common_inherit_sale
+NEW ir.ui.view: sale.sale_order_view_activity
+NEW ir.ui.view: sale.utm_campaign_view_form
+NEW ir.ui.view: sale.utm_campaign_view_kanban
+NEW ir.ui.view: sale.view_sales_order_auto_done_setting
+DEL ir.ui.view: sale.account_invoice_line_form
+DEL ir.ui.view: sale.account_invoice_line_tree
+DEL ir.ui.view: sale.account_invoice_supplier_form
+DEL ir.ui.view: sale.optional_product_items
+DEL ir.ui.view: sale.optional_products_modal
+DEL ir.ui.view: sale.product_configurator_configure
+DEL ir.ui.view: sale.product_configurator_configure_optional_products
+DEL ir.ui.view: sale.product_pricelist_view_form
+DEL ir.ui.view: sale.product_pricelist_view_kanban
+DEL ir.ui.view: sale.product_pricelist_view_tree
+DEL ir.ui.view: sale.report_all_channels_sales_view_pivot
+DEL ir.ui.view: sale.report_all_channels_sales_view_search
+DEL ir.ui.view: sale.sale_onboarding_quotation_layout_form
+DEL ir.ui.view: sale.sale_product_configurator_view_form
+DEL ir.ui.view: sale.variants_tree_view
+DEL ir.ui.view: sale.view_order_form_editable_list
+NEW mail.template: sale.mail_template_sale_confirmation (noupdate)
+NEW res.groups: sale.group_auto_done_setting
```

Comment:
commit https://github.com/odoo/odoo/commit/5d7c0267a68b906afc267a99ab1ae7257bba8f66#diff-1cb1f2dc3616254c38f08837911f00573042c98fb2d0db1cb8e15d6478bc04f0
<pre>
if env['ir.config_parameter'].sudo().get_param('sale.auto_done_setting'):
    # need implied_group sale.group_auto_done_setting
</pre>

Reply:
Good catch! Added in the last commit.